### ISSUE TITLE
JOSS review

### ISSUE THREAD

**Issue by llrs** [llrs/experDesign#32]
Review for JOSS at this issue: https://github.com/openjournals/joss-reviews/issues/3358

Review 1: unmarked sections

- [x] License: Does the repository contain a plain-text LICENSE file with the contents of an OSI approved software license?

This is on file LICENSE.md and LICENSE on the root directory of the package and named on the DESCRIPTION file.  
- [ ] Substantial scholarly effort: Does this submission meet the scope eligibility described in the JOSS guidelines

There are few lines of code (but I won't add lines of code unless they are necessary) and it hasn't been cited but I'm not sure what I can do about the other aspects of [the guide](https://joss.readthedocs.io/en/latest/submitting.html#substantial-scholarly-effort). 

- [x] Example usage: Do the authors include examples of how to use the software (ideally to solve real-world analysis problems).

After reducing the duplicate text, there will be more space available to include some code examples and comparison as explained.

- [x] Functionality documentation: Is the core functionality of the software documented to a satisfactory level (e.g., API method documentation)?

Not sure if I should document it better on the package or on the article. 

- [x] A statement of need: Does the paper have a section titled 'Statement of Need' that clearly states what problems the software is designed to solve and who the target audience is?

Review and add if I missed it, or reword it to make it clearer.

- [ ] Quality of writing: Is the paper well written (i.e., it does not require editing for structure, language, or writing quality)?

I should double check again and probably ask to be reviewed by someone used to check articles. 

Comments on the text: [10.21105.joss.03358.docx](https://github.com/llrs/experDesign/files/6762498/10.21105.joss.03358.docx)
Mostly about not precise language, duplicate text, some parts are not clear (mentioning experDesign on previous work is confusing), and no prove of claims

**Comment by abartlett004** [comment]
Comments for unchecked items on my review checklist:

- Substantial scholarly effort: Does this submission meet the scope eligibility described in the JOSS guidelines

I think this package is useful and I am glad it exists in the world, but it is not a very substantial piece of work. As the JOSS guidelines state, "a minimum allowable contribution should represent not less than three months of work for an individual," which I do not believe is true for this package. I am also not sure what you can do to fix this. 

- A statement of need: Does the paper have a section titled 'Statement of Need' that clearly states what problems the software is designed to solve and who the target audience is?

The paper does a good job of explaining what problems the software is designed to solve, but this is described throughout the paper rather than in one specific section. Also, the target audience is not described in much detail. 

- Quality of writing: Is the paper well written (i.e., it does not require editing for structure, language, or writing quality)?

There were several instances of unclear or misleading phrasing, improper grammar, and informal terminology. I tried to edit or point out many of these, but I think it would be useful if Joe Moore or another experienced proofreader examined the paper again in detail to fully resolve these issues. Relatedly, I think the language in the introduction section can be made more precise such that it can convey its message in just a few paragraphs, which would leave more space for describing the package and comparing it against existing packages in the description section.

**Comment by llrs** [comment]
Many thanks @abartlett004 for commenting here too with such great detail and transparency. 

About the required substantial scholarly effort; I agree that the package is simple, however it took around 3 months of work to create and simplify the package to the actual shape. However, I might try to add some new features from the open issues to make it worth it.

I will add the  statement of need and describe the target audience. I failed to convey that on the manuscript.

The manuscript will be examined and proofread again after the editions to remove informal terminology and clarify or remove unclear and misleading phrasing. Thanks for the suggestions, I'll do my best.

**Comment by llrs** [comment]
I think I fixed all the issues and replied on openjournals/joss-reviews#3358.  
If further modifications are needed I'll reopen the issue
